src/shell/Shell.cpp

```diff
@@ -340,6 +340,12 @@ export class Shell final: public CoreVM::Runtime
         .returnType(CoreVM::LiteralType::Boolean)
         .bind(&Shell::builtinSet, this);
 
+    registerFunction("substitute")
```

Comment:
```suggestion
    registerFunction("getenv")
```
I think that it should be better `getenv` or `variableSubst`; what do you think?

Reply:
Renamed to getenv